src/css/components/_progress.scss

```diff
@@ -17,6 +17,10 @@
   display: none;
 }
 
+.vjs-liveui .vjs-progress-control {
+  @include display-flex(center);
```

Comment:
shouldn't this be `flex-start`?

Reply:
this simply brings back the normal [display value seen above in the file](https://github.com/videojs/video.js/blob/master/src/css/components/_progress.scss#L7). Previously this was hidden during live playback with `display: none` this code makes sure that it is show, but only when the `liveui` class is on the player. Which is added through the `liveui` option.